Lib/System.cpp

```diff
@@ -115,13 +115,16 @@ void handleSignal (int sigNum)
     TERMINAL_SIGNAL_HANDLED = true;
     Shell::reportSpiderFail();
     if(Shell::outputAllowed(true)) {
-      std::cout << getpid() << " Aborted by signal " << signalToString(sigNum);
+      Shell::addCommentSignForSZS(std::cout);
+      std::cout << "Aborted by signal " << signalToString(sigNum);
```

Comment:
Do we perhaps still want the PID to work out which process has received the signal? But no strong feelings, just wondered if this was an oversight.

Reply:
Ahah, my mistake. Thanks!